src/nupic/algorithms/Cells4.cpp

```diff
@@ -188,16 +188,15 @@ bool Cells4::computeUpdate(UInt cellIdx, UInt segIdx, CStateIndexed& activeState
       highWaterSize = segment.size();
       newSynapses.reserve(highWaterSize);
     }
-    for (UInt i = 0; i != segment.size(); ++i)
+    for (UInt i = 0; i < segment.size(); ++i) {
```

Comment:
just for `for` ? shouldn't it be unified? as i see eg in  https://github.com/numenta/nupic.core/pull/278/files#diff-4c232a262b217bd7ab4e516856187881R558 
..anyway, i will change it and we can discuss code-style another time.